### ISSUE TITLE
De website van ATP moet nog vertaald worden naar het Nederlands

### ISSUE THREAD

**Issue by HenriKorver** [VNG-Realisatie/api-test-platform#340]
Nu is de website van het ATP in het Engels. Het moet ook mogelijk zijn te kunnen switchen naar het Nederlands. Het vertaalmechanisme is al ingebouwd maar nog niet ingevuld met Nederlandse teksten. Dit moet nog gebeuren. Bovendien is het een wens van de opdrachtgever dat de default Nederlands is.

**Comment by alextreme** [comment]
Mij lijkt het dan wel verstandig om ook te bedenken hoe om te gaan met teksten en labels zoals via de admin ingevoerd. Zouden die ook tweetalig ingevoerd moeten kunnen worden?

https://api-test.nl/admin/testsession/sessiontype/11/change/

![image](https://user-images.githubusercontent.com/226839/73831563-d7751680-4806-11ea-9e3f-533c902e7922.png)

**Comment by HenriKorver** [comment]
Laten we dit slicen. Dus eerst ervoor zorgen dat we de labels en teksten van de ATP website zelf vertaald kunnen worden naar het Nederlands. Of alle content in de website vertaald kan worden is een zorg voor later en voor nu niet urgent. Het gaat mij erom dat we voldoen aan de eis van het MT van VNG Realisatie dat alle websites en applicaties die gemaakt worden in opdracht van VNG minimaal in het Nederlands zijn.

**Comment by HenriKorver** [comment]
@alextreme en @stevenbal: Is de user story nu duidelijk en kunnen jullie die oppakken?
Volgens mij is het sowieso fijn als het switchen van NL naar ENG gewoon werkt. Want nu wordt deze functionaliteit wel aangeboden door het ATP maar er gebeurt niks als je van ENG naar NL wil switchen:
![afbeelding](https://user-images.githubusercontent.com/37145898/73934483-d7e0e080-48de-11ea-9f90-75c6e3432ec1.png)

**Comment by alextreme** [comment]
Laten we het tijdens het volgend overleg bespreken, het is geen eenvoudig verzoek en gaat aan Steven's kant flink wat tijd kosten.

**Comment by HenriKorver** [comment]
Prima dan bespreken we dit in het volgende overleg. Ik dacht dat het hele vertaal- en switch mechanisme er al in zat en dat het slechts even om het vullen van de vertaaltabel ging. Dus kleine moeite, groot plezier, maar dat is helaas iets te voorbarig. Kunnen jullie wel de "Change language" functionaliteit verbergen want nu krijgt de gebruiker die naar Nederlands wil switchen valse hoop ;-)

**Comment by stevenbal** [comment]
> Laten we dit slicen. Dus eerst ervoor zorgen dat we de labels en teksten van de ATP website zelf vertaald kunnen worden naar het Nederlands. Of alle content in de website vertaald kan worden is een zorg voor later en voor nu niet urgent. Het gaat mij erom dat we voldoen aan de eis van het MT van VNG Realisatie dat alle websites en applicaties die gemaakt worden in opdracht van VNG minimaal in het Nederlands zijn. Wij kunnen niet verantwoordelijk zijn voor de content die een Admin of een gebruiker erin zet.

@HenriKorver ik zal hier vandaag mee aan de gang gaan, ik denk dat het wel wat tijd zal kosten

**Comment by HenriKorver** [comment]
@stevenbal Ik heb dit issue naar de backlog geplaatst en met @alextreme afgesproken dat we het eerst zouden bespreken in een volgend overleg. Het zonde om hier teveel tijd aan te besteden.

**Comment by HenriKorver** [comment]
Het heeft alleen zin om hier aan te werken als de uren die we per maand hebben anders verloren zouden gaan.

**Comment by alextreme** [comment]
Yes, net even besproken met Steven en deze was vanwege een eerdere opmerking per ongeluk al op de agenda van Steven voor vandaag gekomen. De gewenste werkwijze wb ontwikkeluren is nog onduidelijk dus eerst overleggen is het handigst.

**Comment by stevenbal** [comment]
Oke, dan zal ik hem laten voor nu
